notebook/notebook/notebook.py

```diff
@@ -131,10 +171,153 @@ def external_url_for(path):
     return url + path
 
 
+def forbidden():
+    return 'Forbidden', 403
+
+
 @app.route('/healthcheck')
 def healthcheck():
     return '', 200
 
+##### New notebook methods meant for consumption from agnostic client#####
+###### Methods that do not call the auth-gateway /verify method must not be directly public #######
+@app.route('/verify/<svc_name>/', methods=['GET'])
+def validate_get(svc_name):
+    access_token = request.args.get('authorization')
+    token = request.args.get('token')
```

Comment:
Need to double check whether this needs to be escaped.

https://legacy.python.org/dev/peps/pep-0249/#id15

Prepared statements should not need escaping in SELECT context; driver should prevent malicious query vars from performing operations not asked for by the writer of the query, as query form sent separately to db.

https://stackoverflow.com/questions/8263371/how-can-prepared-statements-protect-from-sql-injection-attacks